### PR TITLE
add force_inclusion_values

### PR DESCRIPTION
**Pull request by MRossol**
PR for #230 
- Added force_inclusion_values to Layers class
- Added a new _force_include attr and property
- Isolate force_include layers and run them after regular layers, use np.maximum instead of np.minimum to force include layer values.

### CODE REVIEW

File: reV/supply_curve/exclusions.py [hunk 1]
```diff
@@ -710,6 +730,27 @@ def _generate_ones_mask(self, ds_slice):
...
+    def _force_include(self, mask, layers, ds_slice):
+        """
+        Apply force inclusion layers
+
+        Parameters
+        ----------
+        mask : ndarray | None
+            Mask to apply force inclusion layers to
+        layers : list
+            List of force inclusion layers
```
Comment (grantbuster) [inline reV/supply_curve/exclusions.py:742]: add ds_slice docstring?

**Review by grantbuster — CHANGES_REQUESTED** [review]
Nice this looks great! One minor doc string comment and can you also add the force_include option to the SC agg CLI help string under the excl_dict input?

File: reV/supply_curve/exclusions.py (hunk 1 above)
Comment (MRossol) [reply reV/supply_curve/exclusions.py:742]: whoops

**Comment by MRossol** [comment]
Doc strings updated

**Review by grantbuster — APPROVED** [review]

**Pull request merged**